Setup/Patch/Data/AddShowInXmlSitemapCategoryAttribute.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Aligent Consulting
+ * Copyright (c) Aligent Consulting (https://www.aligent.com.au)
+ */
+
+declare(strict_types=1);
+
+namespace Aligent\Sitemap\Setup\Patch\Data;
+
+use Magento\Catalog\Api\Data\CategoryAttributeInterface;
+use Magento\Eav\Model\Entity\Attribute\Source\Boolean;
+use Magento\Eav\Model\Entity\Attribute\ScopedAttributeInterface;
+use Magento\Eav\Setup\EavSetup;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+use Zend_Validate_Exception;
+
+class AddShowInXmlSitemapCategoryAttribute implements DataPatchInterface
+{
+    /**
+     * @param ModuleDataSetupInterface $moduleDataSetup
+     * @param EavSetup $eavSetup
+     */
+    public function __construct(
+        private readonly ModuleDataSetupInterface $moduleDataSetup,
+        private readonly EavSetup $eavSetup
+    ) {
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public static function getDependencies(): array
+    {
+        return [];
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getAliases(): array
+    {
+        return [];
+    }
+
+    /**
+     * @inheritDoc
+     * @throws LocalizedException
+     * @throws Zend_Validate_Exception
+     */
+    public function apply(): AddShowInXmlSitemapCategoryAttribute
+    {
+        $this->moduleDataSetup->startSetup();
+
+        $attributeSetId = $this->eavSetup->getDefaultAttributeSetId(
+            CategoryAttributeInterface::ENTITY_TYPE_CODE
+        );
+
+        $attributeGroupId = $this->eavSetup->getDefaultAttributeGroupId(
+            CategoryAttributeInterface::ENTITY_TYPE_CODE,
+            $attributeSetId
+        );
+
+        foreach ($this->getShowInXmlSitemapCategoryAttributeConfiguration() as $code => $config) {
```

Comment:
Why are we using a loop when the function only returns a single entry?
The data from the function could just be included here as:
```
$code = 'show_in_sitemap';
$config = [
    'type' => 'int',
    ...
]
```

Or have both values as constants

Reply:
@aligent-lturner Requested changes are done. Please check.